### PR TITLE
feat(core): urn for groupStatus where member is indirect

### PR DESCRIPTION
**Pull request by mattjoke**
* created urn, that returns if the member provided is INDIRECT

### CODE REVIEW

File: perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_groupStatusIndirectTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+	public AttributeDefinition setUpMemberOrgsAttrDef() throws Exception {
+		AttributeDefinition attrDef = new AttributeDefinition();
+		attrDef.setNamespace(AttributesManager.NS_MEMBER_ATTR_DEF);
+		attrDef.setDescription("Member organizations");
+		attrDef.setFriendlyName("memberOrganizations");
+		attrDef.setType(ArrayList.class.getName());
+
+		return perun.getAttributesManagerBl().getAttributeDefinition(sess, A_MEMBER_DEF_MEMBER_ORGANIZATIONS);
+	}
```
Comment (bodnara) [inline perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_groupStatusIndirectTest.java:62]: This is probably not needed?

**Review by bodnara — APPROVED** [review]
Seems good to me.

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_groupStatusIndirect.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+	}
+
+	@Override
+	public AttributeDefinition getAttributeDefinition() {
+		AttributeDefinition attr = new AttributeDefinition();
+		attr.setNamespace(AttributesManager.NS_MEMBER_GROUP_ATTR_VIRT);
+		attr.setFriendlyName("groupStatusIndirect");
+		attr.setDisplayName("Member group indirect status");
+		attr.setType(String.class.getName());
```
Comment (xflord) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_groupStatusIndirect.java:58]:
```suggestion
		attr.setType(Boolean.class.getName());
```

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_groupStatusIndirect.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ * Returns if member is INDIRECT for specified member and group
+ *
+ * @author Matej Hakoš <492968@muni.cz>
+ */
+public class urn_perun_member_group_attribute_def_virt_groupStatusIndirect extends MemberGroupVirtualAttributesModuleAbstract implements MemberGroupVirtualAttributesModuleImplApi {
+
+	final static Logger log = LoggerFactory.getLogger(urn_perun_member_group_attribute_def_virt_groupStatusIndirect.class);
+
+	@Override
+	public void checkAttributeSyntax(PerunSessionImpl perunSession, Member member, Group group, Attribute attribute) throws WrongAttributeValueException {
```
Comment (xflord) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_groupStatusIndirect.java:27]: Decide, do you want to use string or boolean? getAttributeValue treats the attribute as true/false and checkAttributeSyntax as "INVALID"/null.

**Review by xflord — CHANGES_REQUESTED** [review]

**Comment by mattjoke** [comment]
I have tested the urn. The urn successfully returns the indirect relation based on the member id.

**Review by xflord — APPROVED** [review]

**Review by Johaney-s — APPROVED** [review]

**Pull request merged**